package/gluon-mesh-batman-adv/files/lib/netifd/proto/gluon_bat0.sh

```diff
@@ -43,13 +43,7 @@ proto_gluon_bat0_setup() {
 	ip link set primary0 address "$primary0_mac" mtu 1532 up
 
 	local routing_algo="$(lua -e 'print(require("gluon.site").mesh.batman_adv.routing_algo())')"
-
-	if [ "$routing_algo" = 'BATMAN_IV_LEGACY' ]; then
-		modprobe batman-adv-legacy
-	else
-		modprobe batman-adv
```

Comment:
If we autoload the module now, shouldn't we get rid of the `rmmod batman-adv` in `proto_gluon_bat0_teardown`?

Reply:
Fixed